library/core/resource_loader/src/main/resources/quilt_resource_loader.mixins.json

```diff
@@ -16,7 +16,11 @@
     "client.CreateWorldScreenMixin",
     "client.FontManagerResourceReloadListenerMixin",
     "client.GameOptionsMixin",
-    "client.KeyedClientResourceReloaderMixin"
+    "client.KeyedClientResourceReloaderMixin",
+    "client.MinecraftClientMixin"
+  ],
+  "server": [
```

Comment:
No reason to mark the mixin server only as the target always exists on client

Reply:
Uh? The dedicated server main class exists on the client???

Reply:
Yep